src/test/groovy/test/SessionSpec.groovy

```diff
@@ -12,4 +12,42 @@ class SessionSpec extends GebReportingSpec {
 
     def cleanup() {}
 
+    def "Should be visible all fields in form"() {
+        when:
+        to SessionPage
+
+        then:
+        sessionDateInput.isDisplayed()
+        postalCodeInput.isDisplayed()
+        cityInput.isDisplayed()
+        addressInput.isDisplayed()
+        additionalInformationInput.isDisplayed()
+        typeOfSpaceRadioButtons*.isDisplayed()
+        spaceForSession.isDisplayed()
+        levelDropDownIsDisplayed
+        productDropDownIsDisplayed
+        examinerSelectIsDisplayed
+    }
+
+    def "Should be visible all correct titles in form"() {
+        when:
+        to SessionPage
+        def data = CommonHelper.jsonToObject("src/test/resources/applicationData.json")
```

Comment:
To by można przesunąć do bloku given: czyli:
given:
--------
when:
--------
then:
--------